umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -72,5 +73,11 @@
         update-format (or update-format (:format concept))
         umm (spec-core/parse-metadata
              context concept-type format metadata {:sanitize? (= :umm-json (mt/format-key update-format))})
-        umm (apply-update update-type umm update-field update-value find-value)]
+        metadata-dates (:MetadataDates umm)
```

Comment:
I asked @eereiter about this before asking you. He got back to me later saying we need to update this field after bulk update.  He said the difference between the normal update through ingest and bulk update is that the normal update is through someone modifying the metadata file and ingest vs they modify the metadata file through our bulk update code. so we always have to update the MetadataDates/UPDATE - add it if it doesn't exist.   Now my only question is I'm not sure if we should keep multiple UPDATE field, or just having one latest one.